saleor/order/__init__.py

```diff
@@ -96,6 +96,7 @@ class OrderEvents:
     FULFILLMENT_FULFILLED_ITEMS = "fulfillment_fulfilled_items"
     TRACKING_UPDATED = "tracking_updated"
     NOTE_ADDED = "note_added"
+    CUSTOMER_NOTE_ADDED = "customer_note_added"
```

Comment:
We did not add a new note, we simply edited the customer note
```suggestion
    CUSTOMER_NOTE_UPDATED = "customer_note_updated"
```